bids/modeling/statsmodels.py

```diff
@@ -343,6 +342,15 @@ def _build_groups(objects, group_by):
         entities = [obj.entities for obj in objects]
         df = pd.DataFrame.from_records(entities)
 
+        # Load participants.tsv as entities
+        if self.level == 'dataset':
+            # Separate BIDSVariableCollections
+            collections = [o.to_df() for o in objects if type(o).__name__ == 'BIDSVariableCollection']
```

Comment:
would this work if participants.tsv does not exist?

Reply:
Yes. If it doesn't exist then the collection will either be missing or empty.